package.json

```diff
@@ -30,6 +30,7 @@
     "hubot-matteruser": "github:contolini/hubot-matteruser#fix-v1-dep",
     "hubot-murder": "^1.0.0",
     "hubot-new-relic-alerts": "^1.0.0",
+    "hubot-newrelic2-cfpb": "https://github.com/cfpb/hubot-newrelic2-cfpb.git",
```

Comment:
FYI npm's resolver defaults to github so you can do the shorthand `cfpb/hubot-newrelic2-cfpb` instead of the full path.